stack/rush-stack/includes/tsconfig-node.json

```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+  "extends": "../../rush-stack-compiler/includes/tsconfig-node.json"
```

Comment:
>../../rush-stack-compiler [](start = 14, length = 25)

Seems like this may not point to the right location, depending on how the package manager chooses to install **rush-stack**.  Possible options:

1. Improve the compiler to support NPM resolution like what **tslint.json** does.  This would need to be done for any other config files tied to **rush-stack-compiler**

2. Use a `postInstall` script to perform NPM resolution, and write .json files into the **rush-stack-compiler** folder to perform redirection.

3. Keep it simple:  Have the project's tsconfig.json extend from a direct dependency (e.g. **rush-stack-compiler** or **rig-compiler**).  If the **rig-compiler** wants to inherit from **rush-stack-compiler**, use a build step to make a static copy of the file contents, rather than trying to follow multiple hops through the node_modules folder.

I think # 3 might be the safest bet, although the other options are interesting.